plotting/map.py

```
@@ -344,6 +344,7 @@ def load_data(self):
 
         if self.compare:
             self.variable_name += " Difference"
+            self.cmap = cmap = colormap.find_colormap(self.compare.get('colormap_diff'))
```

Ruff (0.12.2)

Local variable `cmap` is assigned to but never used

Remove assignment to unused variable `cmap`

(F841)